tests/e2e/device-plugin_test.go

```diff
@@ -144,6 +145,8 @@ var _ = NVDescribe("GPU Device Plugin", func() {
 			It("it should run GPU jobs", func(ctx context.Context) {
 				By("Creating GPU job")
 				job := common.GPUJob.DeepCopy()
+				runtimeClassName := "nvidia"
+				job.Spec.Template.Spec.RuntimeClassName = &runtimeClassName
```

Comment:
Instead of doing this, you can have a function defined as:
```
func ptr[T any](v T) *T {
  return &v
}
```
And then just do this here:
```
job.Spec.Template.Spec.RuntimeClassName = ptr("nvidia")
```
Probably not worth it if this pattern is only used once (I haven't checked), but worthwhile if used alot.

Reply:
This is used elsewhere in our tests. Definitely useful. I will look at making it accessible. 

As a matter of interest, is there guidance on where this kind of functionality should be implemented? (`test-utils`?)

Reply:
I think for now makes sense to have on the device-plugin file. or we will use this also for the gfd tests? if so, then utils makes sense